crates/nu-cli/src/commands/autoview.rs

```diff
@@ -234,8 +247,10 @@ pub fn autoview(context: RunnableContext) -> Result<OutputStream, ShellError> {
                                 yield Err(e);
                             }
 
-                            Value { value: UntaggedValue::Row(row), ..} if !no_auto_pivot
-                                || (pivot_to_fit && // Or if the row character count + number of headers * 2 (for padding) > terminal width
+                            Value { value: UntaggedValue::Row(row), ..} 
+                                if pivot_mode == AutoPivotMode::Always || 
+                                ((pivot_mode == AutoPivotMode::ToFit || 
+                                    pivot_mode == AutoPivotMode::Auto) &&
```

Comment:
This makes me think `ToFit` and `Auto` are the same. What do you think?

Reply:
I tried to implement it as you instructed.
```
I'm also thinking we should collapse the multiple config 
values into one: pivot_mode with values auto, always, 
never, to_fit (perhaps to_fit could be the auto)
```
So, `to_fit` is the `auto`.

Reply:
oh, perhaps you're saying to rename `to_fit` to `auto`. If that's what you mean, i can change it.

Reply:
Yep, you got it 🙂